Jenkinsfile

```diff
@@ -30,49 +31,94 @@ pipeline {
         IQE_MARKER_EXPRESSION="compliance_smoke"
         IQE_PLUGINS="compliance"
         REF_ENV="insights-stage"
+
+        // CICD_TOOLS_URL='https://raw.githubusercontent.com/RedHatInsights/cicd-tools/main/src/bootstrap.sh'
     }
 
     stages {
-
-        stage('Build the PR commit image') {
-            steps {
-                withVault([configuration: configuration, vaultSecrets: secrets]) {
-                    sh 'bash -x build_deploy.sh'
-                }
-            }
-        }
-
         stage('Run Tests') {
             parallel {
-                stage('Run unit tests') {
+                stage('Build the PR commit image') {
                     steps {
                         withVault([configuration: configuration, vaultSecrets: secrets]) {
-                            sh 'bash -x ./scripts/unit_test.sh'
+                            sh '''
+                            env
+                            
+                            IMAGE_TAG=$(git rev-parse --short=7 HEAD)
```

Comment:
I'd suggest to remove this line, as this is already set by the `build_image` function, see [build_deploy.sh](https://github.com/RedHatInsights/compliance-backend/blob/1b1162028463288e96e0668f0c657a0385d08935/build_deploy.sh#L23)